lib/libc/string/wcsncmp.c

```diff
@@ -43,16 +43,12 @@ __FBSDID("$FreeBSD$");
 int
 wcsncmp(const wchar_t *s1, const wchar_t *s2, size_t n)
 {
-
```

Comment:
All the same comments as before: gratuitous changes and regresses wchar_t-is-int architectures